src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts

```diff
@@ -241,13 +241,10 @@ export const activateTribunalOrdersAndRequestsLink = (
   }
 };
 
-export const filterNotificationsWithRequestsOrOrders = (
-  items: SendNotificationTypeItem[]
-): SendNotificationTypeItem[] => {
+export const filterSendNotifications = (items: SendNotificationTypeItem[]): SendNotificationTypeItem[] => {
   return items?.filter(
     it =>
-      it.value.sendNotificationCaseManagement !== undefined &&
-      it.value.sendNotificationCaseManagement !== '' &&
-      it.value.sendNotificationCaseManagement !== null
+      it.value.sendNotificationSubjectString === NotificationSubjects.ORDER_OR_REQUEST ||
+      it.value.sendNotificationSubjectString === NotificationSubjects.GENERAL_CORRESPONDENCE
```

Comment:
I'm not sure this logic works in all cases

`sendNotificationSubjectString` is a single string representing `join`'d subjects. So while only selecting "Other (General correspondence)" in ExUI will result in the string "Other (General correspondence)", selecting that AND hearings results in "Hearings, Other (General correspondence)". 

I'm aware there has been some confusion between actual usage patterns and what's technically possible, have you spoken to BAs about this possibility? ie, what do we if multiple subjects are select (one of which is Other)

Reply:
@JackReevies Didn't realise you could do that and I don't its likely we get a change request for it anytime soon. I've modified the filtering code so it checks if either of titles exists in the string